R/generate_template.R

```diff
@@ -313,3 +345,53 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
 
     return()
 }
+
+create_default_theme_file <- function(theme_file) {
+    lines <- c("### primary_color",
+               "# To set primary status color, that affect the color of header, valueBox, infoBox and box",
+               "# Valid possible value are colors names or hex-decimal value (i.e,: \"blue\", \"#086A87\")",
+               "# Blank/empty value will use default value",
+               "primary_color: \n\n",
+               
+               
+               "# Sidebar variables: change the default sidebar width, colors:",
+               "### sidebar_width",
+               "# width is numeric value in pixels, must be greater than 0",
+               "# Valid possible value are 200, 350, 425, ...",
+               "# Blank/empty value will use default value",
+               "sidebar_width: \n",
+               
+               "### sidebar_background_color",
+               "# Valid possible value are colors names or hex-decimal value (i.e,: \"blue\", \"#086A87\")",
+               "# Blank/empty value will use default value",
+               "sidebar_background_color: \n",
+               
+               "### sidebar_item_background_hover_color",
+               "# Valid possible value are colors names or hex-decimal value (i.e,: \"blue\", \"#086A87\")",
```

Comment:
Can you explain the purpose of this field. In addition to specifying the valid values

Reply:
done, and possible values are already there